openedx/core/djangoapps/discussion_common/comment_client/thread.py

```diff
@@ -112,19 +113,23 @@ def search(cls, query_params):
         )
 
     @classmethod
-    def url_for_threads(cls, params={}):
-        if params.get('commentable_id'):
-            return u"{prefix}/{commentable_id}/threads".format(prefix=settings.PREFIX, commentable_id=params['commentable_id'])
+    def url_for_threads(cls, params=None):
+        if params and params.get('commentable_id'):
+            return u"{prefix}/{commentable_id}/threads".format(
+                prefix=settings.PREFIX,
+                commentable_id=params['commentable_id'],
+            )
         else:
             return u"{prefix}/threads".format(prefix=settings.PREFIX)
 
     @classmethod
-    def url_for_search_threads(cls, params={}):
+    def url_for_search_threads(cls, params=None):
         return "{prefix}/search/threads".format(prefix=settings.PREFIX)
```

Comment:
Nit: Can we just get rid of params? It's never used...?

Reply:
I tried to be safe and not make such changes.  But since there's only 1 caller of this, ok, sure.